lib/tests/formulae.rb

```diff
@@ -295,7 +295,9 @@ def setup_formulae_deps_instances(formula, formula_name, args:)
         @unchanged_build_dependencies = build_dependencies - @formulae
 
         # Test reverse dependencies for linux-only formulae in linuxbrew-core.
-        if args.keep_old? && formula.requirements.exclude?(LinuxRequirement.new)
+        if tap.full_name == "Homebrew/linuxbrew-core" &&
```

Comment:
This doesn't really work:
```
==> Determining dependencies...
Error: undefined method `full_name' for nil:NilClass
```
https://github.com/Homebrew/brew/pull/11628/checks?check_run_id=2954843691#step:4:25

Did you mean something like this:
```suggestion
        if formula.tap.full_name == "Homebrew/linuxbrew-core" &&
```

Reply:
Will be fixed by https://github.com/Homebrew/homebrew-test-bot/pull/627